examples/full-jaffle-shop-demo/dbt/models/tracks.yml

```diff
@@ -74,9 +74,20 @@ models:
         meta:
           dimension:
             type: string
-          metrics: 
+          metrics:
             unique_event_count:
               type: count_distinct
+              description: |
```

Comment:
I think we can keep this, wdyt @almeidabbm ?

Reply:
Yeah let's keep this